From 495fb9ca8d6689ca61c96962bacfa7e626a1bf92 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Thu, 29 Jan 2015 10:39:51 +0100
Subject: [PATCH 31/57] build: build the sandbox with PIE

(Position Independent Executable.)
This patch originally from openSUSE, chromium-sandbox-pie.patch.
---
 sandbox/linux/sandbox_linux.gypi | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/sandbox/linux/sandbox_linux.gypi b/sandbox/linux/sandbox_linux.gypi
index f5b3e0f..f57fbf3 100644
--- a/sandbox/linux/sandbox_linux.gypi
+++ b/sandbox/linux/sandbox_linux.gypi
@@ -205,7 +205,10 @@
       ],
       'cflags': [
         # For ULLONG_MAX
-        '-std=gnu99',
+        '-std=gnu99 -fPIE',
+      ],
+      'ldflags': [
+        '-pie',
       ],
       'include_dirs': [
         '../..',
-- 
2.6.2

